.vscode/launch.json

```diff
@@ -21,7 +21,7 @@
             "type": "coreclr",
```

Comment:
Added these, since was forgetting to rebuild and was feeling stupid that "nothing works", I say we leave them, since I wanter to add them to the tasks for a long time.